src/clarity-angular/emphasis/alert/_alert.clarity.scss

```diff
@@ -81,30 +77,70 @@ $clr-alert-colors: (
     @else {
         border: none;
     }
+
+    .alert-icon {
+        color: map-get($propMap, icon-color)
+    }
 }
 
 @include exports('alert.clarity') {
-    %icon-styles {
+    $alertIconWidth: $clr_baselineRem_1 + 1;
```

Comment:
I'm counting the days until CSS grid is widely adopted. This whole task would've been much easier with CSS grid.